frontend/packages/ceph-storage-plugin/integration-tests-cypress/tests/active-health-checks.spec.ts

```diff
@@ -0,0 +1,73 @@
+import { checkErrors } from '../../../integration-tests-cypress/support';
+import { scaleDeployments } from '../common-operations';
+import { MINUTE } from '../consts';
+
+const enum Deployments {
+  ROOK_CEPH_MON_A = 'rook-ceph-mon-a',
+  ROOK_CEPH_MON_B = 'rook-ceph-mon-b',
+  ROOK_CEPH_MGR_A = 'rook-ceph-mgr-a',
+}
+
+const messages = {
+  warnings: {
+    MON_DOWN: '1/3 mons down, quorum b,c',
+    MGR_DOWN: 'no active mgr',
+  },
+  errors: {
+    ERROR: 'failed to get status. . timed out: exit status 1',
```

Comment:
I've kept this to ERROR as `error` is what we get from the JSON response from the `ceph` object and so that it complies with [this](https://github.com/openshift/console/pull/9690/files#diff-fa20d6bd8341e9a45df0d338a1cee9a6490b5f86dfe10f011c4c6e03b61c63b2R43). Please refer the [source](https://github.com/openshift/console/blob/110499002d84fe9a24c2533c96edd86e4466029c/frontend/packages/ceph-storage-plugin/src/components/dashboards/persistent-internal/status-card/status-card.tsx#L106) for how we catch these (notice that we have only a single straightforward `error` in the RegEx).